docs/examples.md

```diff
@@ -0,0 +1,38 @@
+# Examples
+
+!!! Tip "Logfire playground"
+Example projects provide options to try out Logfire without needing to instrument your app in production.
+
+These are working, stand-alone apps and projects that you can clone, spin up locally and play around with to get a feel for the different capabilities of Logfire.
+
+**Got a suggestion?**
+
+If you want to see an example of a particular language or library, [get in touch](help.md).
+
+# Example repositories on GitHub
+
+## JavaScript
+
+### Cloudflare worker example
+
+This example is based on the scaffolding created from `npm create cloudflare@latest`, and uses the [otel-cf-workers package](https://github.com/evanderkoogh/otel-cf-workers) to instrument a Cloudflare Worker and send traces and metrics to Logfire.
+
+**Check it out on github:**
+`https://github.com/pydantic/logfire/tree/main/examples/javascript/cloudflare-worker/`
```

Comment:
ok so this is super annoying - can't have external links in mkdocs apparently? I had a quick look for work arounds but didn't make progress, if anyone has a nice fix for this, putting absolute links in as code is definitely not my preferred solution here. 

![Screenshot 2024-09-26 at 10 00 14](https://github.com/user-attachments/assets/4d9ae0e3-5f52-45dd-b33c-288a3d7186b9)


https://www.mkdocs.org/user-guide/writing-your-docs/

Reply:
I don't understand the problem. What can't you do?

Reply:
@Kludex I tried making absolute URLs to the github link (rather than making it an ugly code link) and the docs refused to compile and the link wouldn't work. Hmm. I wonder if the problem is not the external link, but rather that those directories don't actually exist yet on GH because they are created by this PR 🤔

Reply:
what's the error you have? or can you push the code with the issue so I can just try to run the docs?